### DISCUSSION TITLE
Ability to refresh protobuf definitions

### DISCUSSION THREAD

**Discussion by dselans** [Kong/insomnia#4484] [Ideas]
Proto files get updated often as new rpc methods are defined and existing ones are updated.

There is currently no way to "update" these definitions without adding "new" proto definitions. This causes insomnia to clear all body payloads, which means that every single grpc request must be essentially reworked from scratch.

This is a bug report + feature request.

**Comment by Nikhil1920** [comment]
This feature is planned https://docs.insomnia.rest/insomnia/grpc#:~:text=In%20future%20iterations%20we%20intend%20to%20extend%20the%20ways%20in%20which%20you%20can%20input%20a%20proto%20file%2C%20including%20keeping%20in%20sync%20with%20the%20file%20system%2C%20loading%20from%20a%20network%20location
